### PR TITLE
Add release notes for 4.1.0

### PR DESCRIPTION
**Pull request by martyngigg**
**Description of work.**

Adds skeleton release notes for the next release.

**To test:**

Code review and check the builds have no warnings.

*There is no associated issue.*

*This does not require release notes* because **it is the release notes!**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: tools/release_generator/release.py [hunk 1]
```diff
@@ -71,7 +68,7 @@
    Framework <framework>
    User Interface & Usability <ui>
    Diffraction <diffraction>
-   LowQ <lowq>
+   SANS <sans>
```
Comment (AndreiSavici) [inline tools/release_generator/release.py]: So what happens to reflectometry?

Comment (martyngigg) [reply tools/release_generator/release.py]: Good point. I was just following our current layout. It looks like we actually have separate pages for SANS & Reflectometry currently so I think I'll go with that.

**Comment by martyngigg** [comment]
@AndreiSavici I have moved a couple of things around. What do you think of the layout now?

File: docs/source/release/v4.1.0/index.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  `doi: 10.1016/j.nima.2014.07.029 <https://doi.org/10.1016/j.nima.2014.07.029>`_
+  (`download bibtex <https://raw.githubusercontent.com/mantidproject/mantid/master/docs/source/mantid.bib>`_)
+
+
+Changes
+-------
+
+.. toctree::
+   :hidden:
+
```
Comment (AndreiSavici) [inline docs/source/release/v4.1.0/index.rst]:
```suggestion
   :glob:
```

File: docs/source/release/v4.1.0/index.rst [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+   *.rst
```
Comment (AndreiSavici) [inline docs/source/release/v4.1.0/index.rst]:
```suggestion
   *
```

**Review by AndreiSavici — CHANGES_REQUESTED** [review]
You have a lot of docs warnings. I've suggested changes. Need to add `:glob:` to the toctree, and it does not find `*.rst`. For me it works with `*`. I am using sphinx 1.6.7

**Comment by martyngigg** [comment]
Thanks, Andrei. I'll fix that `:glob:` warning. Pete has fixed the `*.rst`

**Comment by peterfpeterson** [comment]
I thought I had fixed it all locally then merged `release-next` into `master` and pushed the result.

**Comment by martyngigg** [comment]
Cheers!

**Pull request merged**